runtime/doc/options.txt

```diff
@@ -4320,20 +4319,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	reset this option. |-u| |--noplugin|
 
 						*'macatsui'* *'nomacatsui'*
-'macatsui'		boolean	(default on)
-			global
-			{only available in Mac GUI version}
-	This is a workaround for when drawing doesn't work properly.  When set
-	and compiled with multi-byte support ATSUI text drawing is used.  When
-	not set ATSUI text drawing is not used.  Switch this option off when
-	you experience drawing problems.  In a future version the problems may
-	be solved and this option becomes obsolete.  Therefore use this method
-	to unset it: >
-		if exists('&macatsui')
-		   set nomacatsui
-		endif
-<	Another option to check if you have drawing problems is
-	'termencoding'.
+'macatsui'		Removed. |vim-differences| {Nvim}
```

Comment:
Why would we want to keep a stub of this legacy option around? It's the first time I see this being done.